pending/xnview.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2008-2018 Andrew Ziem
+    https://www.bleachbit.org
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	CleanerML written by https://github.com/Tobias-B-Besemer
+	v0.1 done on 2018-12-19
+
+-->
+<cleaner id="xnview.new">
+  <label>XnView Classic for Windows</label>
+  <description>Picture Viewer</description>
+  <option id="mru">
+    <label>MRU Files and Folders</label>
+    <description>Delete list of MRU (Most recently used) Files and Folders</description>
+        <action command="ini" search="glob" path="$APPDATA\XnView\xnview.ini" name="FileName_0"/>
```

Comment:
Change all the `glob` to `file` where there is not any wildcards in the filename

```suggestion
        <action command="ini" search="file" path="$APPDATA\XnView\xnview.ini" name="FileName_0"/>
```